src/mon/ConfigKeyService.cc

```diff
@@ -226,3 +253,20 @@ void ConfigKeyService::test_do_thing()
   bl.append("value");
   store_put("test/item", bl);
 }
+
+void ConfigKeyService::do_osd_destroy(int32_t id, uuid_d& uuid)
+{
+  string dmcrypt_prefix =
+    "dm-crypt/osd/" + stringify(uuid) + "/";
+  string daemon_prefix =
+    "daemon-private/osd." + stringify(id) + "/";
+
+  vector<string> prefixes = { dmcrypt_prefix, daemon_prefix };
+
+  MonitorDBStore::TransactionRef t = paxos->get_pending_transaction();
+  for (auto &p : prefixes) {
```

Comment:
fwiw you can do a literal list here, e.g.
<pre>
for (auto p : { dmcrypt_prefix, daemon_prefix }) {
   ...
</pre>